src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -367,16 +368,38 @@ export class SbbFormField implements ComponentInterface {
    */
   private _onSlotErrorChange(event: Event): void {
     this._errorElements = (event.target as HTMLSlotElement).assignedElements();
+
+    for (const el of this._errorElements) {
+      // Although a form error assigns an id itself, we need to be earlier by creating one here
+      if (!el.id) {
+        el.id = `sbb-form-field-error-${++nextFormFieldErrorId}`;
+      }
+      if (!el.role) {
+        // Instead of defining a container with an aria-live region as expected, we had to change
+        // setting it for every slotted element to properly work in all browsers and screen reader combinations.
+        el.role = 'status';
+      }
+    }
     this._applyAriaDescribedby();
     toggleDatasetEntry(this._element, 'hasError', !!this._errorElements.length);
   }
 
   private _applyAriaDescribedby(): void {
-    const value = this._errorElements.length
-      ? this._errorElements.map((e) => e.id).join(' ')
-      : this._originalInputAriaDescribedby;
-    if (value) {
-      this._input?.setAttribute('aria-describedby', value);
+    const ids = [];
+
+    if (this._originalInputAriaDescribedby) {
```

Comment:
Just a quick question: 
in the original code the _originalInputAriaDescribedby OR the joined id are used, while now they can both be used. 
It is correct?

Reply:
Yes I compared to Angular implementation and it make sense in my mind as well that whatever original aria-describedby is never removed

Reply:
Do you have concerns?

Reply:
No, not at all, i just saw the difference so I asked :) can be merged for me